source/content/terminus/install.md

```diff
@@ -29,6 +29,7 @@ There are several ways to install Terminus, depending on your use case:
 
 - For a self-contained Terminus executable, [install terminus.phar](#standalone-terminus-phar).
 - For a Composer-managed version of Terminus that is _not_ part of your other Composer-managed project(s) and doesn't utilize global Composer installations, use the [Terminus installer PHAR](#terminus-installer-phar).
+- If you are in Mac, you could [install using homebrew](#homebrew-installation).
```

Comment:
In addition to adding the homebrew instructions, we should also remove the unofficial installer note below, and the Terminus Installer PHAR as well.

Let's keep the standalone Terminus PHAR instructions, though.

Reply:
I pushed a commit implementing my suggestion above